Web_Development/FrontEnd/HTML_Lists/Readme.md

```diff
@@ -0,0 +1,504 @@
+# **HTML LISTS**
+<div align="center">
+<img src="https://designzonic.com/knowledge-base/wp-content/uploads/2019/05/Beyond-bland_HTML-lists-with-style.png"/>
+</div>
+
+<br>
+
+## What are LISTS ?
+<hr>
+
+- Lists are the preferred way to display items one after the other, instead of just using ```<br>``` tags.
+
+- HTML lists allow us to group a set of related items in lists.
+
+- Lists are very helpful in conveying a set of either numbered or bullet points.
+
+- Lists are the formatting tool for the content of our web pages. 
+
+- HTML lists are good from a structural point of view as they help create a well-structured, more accessible, easy-to-maintain document.
+
+<br>
+<br>
+
+## Types of Lists in HTML
+
+<hr>
+
+There are three types of lists in HTML, which are :
+
+- <span style="color:lightsalmon;">***UnOrdered List***</span> — used to create a list of related items, in no particular order.
+
+- <span style="color:lightsalmon;">***Ordered List***</span> — used to create a list of related items, in a specific order.
+
+- <span style="color:lightsalmon;">***Definition List***</span> — used to create a list of terms and their descriptions.
+
+<br>
+
+**Note :** We can create a list inside another list, which will be termed as *Nested List*.
+
+<br>
+<br>
+
+# UnOrdered or Bulleted Lists in HTML
+
+An *Unordered lists* are used when a set of items can be placed in any order. The list items in unordered lists are marked with bullets. An example is a list of programming languages :
+
+- C++
+- Java
+- Python
+- Ruby
+
+<br>
+
+**Example :**
+
+<br>
+
+<div align="center">
+<img src="https://i0.wp.com/www.tutorialbrain.com/wp-content/uploads/2019/01/Unordered-List.jpg?fit=474%2C397&ssl=1" /> 
+</div>
+
+<br>
+
+## Syntax For UnOrdered List
+
+<hr>
+
+- We use the `<ul>` tag to create an unordered list.
+
+- Items will appear with bullets.
+
+- The list items are created with the `<li>` tag.
+
+<br>
+
+**For Example :**
+
+      <ul>
+          <li>Entry 1</li>
+          <li>Entry 2</li>
+          <li>Entry 3</li>
+      </ul>
```

Comment:
Could be a code-block with syntax.

Reply:
I'm not exactly  getting what you are trying to say ,can you please elaborate.